tools/ndocs-project/Menu.txt

```diff
@@ -67,13 +67,14 @@ Group: Functions  {
    Group: Ai  {
 
       File: CBA_fnc_addWaypoint  (ai/fnc_addWaypoint.sqf)
+      File: CBA_fnc_buildingPositions  (ai/fnc_buildingPositions.sqf)
       File: CBA_fnc_clearWaypoints  (ai/fnc_clearWaypoints.sqf)
       File: CBA_fnc_searchNearby  (ai/fnc_searchNearby.sqf)
       File: CBA_fnc_taskAttack  (ai/fnc_taskAttack.sqf)
       File: CBA_fnc_taskDefend  (ai/fnc_taskDefend.sqf)
       File: CBA_fnc_taskPatrol  (ai/fnc_taskPatrol.sqf)
       File: CBA_fnc_taskSearchArea  (ai/fnc_taskSearchArea.sqf)
-      File: CBA_fnc_waypointGarrison  (ai/fnc_waypointGarrison.sqf)
+      File: fnc_waypointGarrison  (ai/fnc_waypointGarrison.sqf)
```

Comment:
Why is this one different?

Reply:
It's because in `File: XYZ` line, XYZ is a string, generated from header in a files, eg.
 `addons/ai/fnc_waypointGarrison.sqf`
```
/* ----------------------------------------------------------------------------
Script: fnc_waypointGarrison
```

Notice that for example
`addons/ai/fnc_taskAttack.sqf`
```/* ----------------------------------------------------------------------------
Function: CBA_fnc_taskAttack
```
has a proper header, so ndocs generated `File: CBA_fnc_taskAttack` for it.

It's just a matter of fixing `Function` headers inside of fnc_ files

Reply:
Ah! In that case, let's fix the function headers instead. :)

Reply:
Fixed headers in https://github.com/CBATeam/CBA_A3/pull/1211/commits/a467239b80af3bfa8b16e8975fc05b57166068f3 and https://github.com/CBATeam/CBA_A3/pull/1211/commits/3473b0f51d70042923a87e99791632f34bcc07f0